src/components/PageElements/Elements/ConsolidateReviewDecision.tsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react'
+import { SummaryViewWrapperProps } from '../../../formElementPlugins/types'
+import { ApplicationResponse, ReviewResponse } from '../../../utils/generated/graphql'
+import ApplicantResponseElement from './ApplicantResponseElement'
+import ReviewResponseElement from './ReviewResponseElement'
+import strings from '../../../utils/constants'
+import { Icon } from 'semantic-ui-react'
+
+interface ConsolidateReviewDecisionProps {
+  applicationResponse: ApplicationResponse
+  summaryViewProps: SummaryViewWrapperProps
+  isActiveReviewResponse: boolean
+  isNewApplicationResponse: boolean
+  /* can add check for isNewReviewResponse */
+  reviewResponse?: ReviewResponse
+  originalReviewResponse?: ReviewResponse
+  showModal: () => void
+}
+const ConsolidateReviewDecision: React.FC<ConsolidateReviewDecisionProps> = ({
+  applicationResponse,
+  summaryViewProps,
+  isActiveReviewResponse,
+  isNewApplicationResponse,
+  reviewResponse,
+  originalReviewResponse,
+  showModal,
+}) => {
+  const isConsolidation = true
+  const decisionExists = !!reviewResponse?.decision
+
+  // After consolidation is submitted, reviewResponses are trimmed if they are not changed duplicates
+  // or if they are null, we only want to show reviewResponses that are linked to latestOriginalReviewResponse - from reviewer
+  const isNewConsolidation = originalReviewResponse?.id !== reviewResponse?.reviewResponseLinkId
+
+  return (
+    <div>
+      {/* Application Response */}
+      <ApplicantResponseElement
+        applicationResponse={applicationResponse}
+        summaryViewProps={summaryViewProps}
+      />
+      {/* Consolidation Response */}
+      {!isNewConsolidation && reviewResponse && (
```

Comment:
(not required change): although it's pretty easy to see now, but would be great to have this one as variable:
`canShowConsolidationResponse`

Reply:
I think this is touching the "Changes requested" area so I'm a little confused since I can't really test it right now.
I had removed this check because of the previous [comment](https://github.com/openmsupply/application-manager-web-app/pull/695#discussion_r632237408) and I think this is already covered. The `ReviewResponseElement` would return `null` in case this review haven't got a decision done, which should be the case when a **New Review** is done (after changes requested by lvl 2 reviewer) and the consolidator does the re-review. Right?

Reply:
thisReviewLatestResponse => will exists even if the decision is not made. I think this check needs to be isActiveReviewResponse && decisionExists ?

Reply:
Note: `ReviewResponseElement` would return null in case this review haven't got a decision done